### PR TITLE
Add work and About  section 

### PR DESCRIPTION
**Pull request by masangana**
**This PR is about the fourth day project. Add The work and about section**
### Description
- Adding my work section
- Adding about me section
- Implementing the button interactions
- Using Css and Html
- Using Css Grid and flexbox
- Using Github
- stick to the design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,71 +30,264 @@
...
+            </li>
+        </ul>
+    
+        <!--illustrations-->
+        <div class="image-container"><br /></div>
+    </main>
+    
+    <!--works section-->
+    <section class="works-section">
+        <h1 class="works-section-header">My Recent Works</h1>
```
Comment (youmari) [inline index.html:99]: you are not using the right font in this header you should use `font-family: 'Crete Round'` so please try to do so

File: index.html [hunk 2]
```diff
@@ -30,71 +30,264 @@
...
+        <hr />
+        <!--main grid-->
+        <div class="works-section-main-grid">
```
Comment (youmari) [inline index.html:102]: you have learned about grid CSS so this is your opportunity to practice what you've learned about grid CSS, so please try to use it here `display: grid;`

File: index.html [hunk 3]
```diff
@@ -30,71 +30,264 @@
...
+            <h2 class="works-section-project-header">Multi-Post Stories</h2>
+            <p class="work-project-para">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+              Iusto atque possimus, repellendus recusandae dicta obcaecati minima 
+              expedita vero quidem et dolore impedit. Lorem ipsum dolor sit amet 
+              consectetur adipisicing elit. Iusto atque possimus, repellendus 
+              recusandae dicta obcaecati minima expedita vero quidem et dolore 
+              impedit. 
+            </p>
+            <ul class="language-list-body">
```
Comment (youmari) [inline index.html:116]: please try to remove the border so they can be the same as the ones in the Figma design 

- [ ] Your design 
image

- [ ] Figma design 
image

File: index.html [hunk 4]
```diff
@@ -30,71 +30,264 @@
...
+        <p class="about-description">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+        </p>
+        <button type="button" class="about-button button">Get My Resume</button>
+        </div>
+        <div class="about-illustration">
+        <img src="./ressources/illustration-about-me-mobile.png" alt="img">
```
Comment (youmari) [inline index.html]: this illustration is just for design purposes so please try to use it as a background image with CSS

File: index.html [hunk 5]
```diff
@@ -30,71 +30,264 @@
...
+            <button type="button" class="grid-project-button button">See Project</button>
+          </div>
+        </div>
+    </section>
+    <!--end-->
+    <!--About me-->
+    <section class="about">
+    <div class="about-section">
+        <div class="about-sub-section">
+        <h1 class="about-header header">About me</h1>
```
Comment (youmari) [inline index.html:235]: you are not using the right font in this header you should use `font-family: 'Crete Round'` so please try to do so

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @masangana , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ Great README file 
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
Hi @masangana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**